updates/handlers.go

```diff
@@ -59,7 +59,8 @@ func (update MediaUpdate) handle() error {
 	if update.Update.FromChat().IsPrivate() {
 		return nil
 	}
-	if strings.ToLower(update.Update.Message.Caption) == "skip" {
+	lines := strings.Split(strings.ToLower(update.Update.Message.Caption), "\n")
+	if len(lines) > 0 && strings.ReplaceAll(lines[0], " ", "") == "skip" {
```

Comment:
The first part of this statement is always true if there's a caption.
IMHO a more readable code would check `if caption != ""` before this line.
The second part I'm not sure is necessary because if you have a link break that would normally won't be after, or cause a space... In anycase if this is just for better handling of messages it makes sense..

Reply:
As you suggested, I'll fix the condition to check it for an empty string beforehand. 

The second part is for occasions where someone typed: "skip  "  (with spaces) and then dropped a line - I can see myself doing that, so I wish to clean all spaces from the intended skip.